CoreScriptsRoot/Modules/Settings/Pages/Players.lua

```diff
@@ -117,12 +134,42 @@ local function Initialize()
 					if friendLabel and friendLabelText and friendLabelText.Text ~= '' then
```

Comment:
Seems like some of the logic in here could be moved out of this script and we could add the same functionality when trying to add a friend in the PlayerList.